sites/royalroad.py

```diff
@@ -37,8 +37,10 @@ def extract(self, url):
         for chapter in soup.select('#chapters tbody tr[data-url]'):
             chapter_url = str(urllib.parse.urljoin(story.url, str(chapter.get('data-url'))))
 
+            # Have to get exact publishing time from the chapter page
+            chapter_soup = self._soup(chapter_url)
```

Comment:
With the extra fetch, might be saner to just make `_chapter` return either a tuple of `contents, updated`... or just the entire `Chapter` object. Thoughts?